src/js/components/Layer/README.md

```diff
@@ -237,6 +237,16 @@ Defaults to
 undefined
 ```
 
+**layer.container.extend**
+
+Any additional style for Layer inner Container. Expects `string | (props) => {}`.
```

Comment:
what does "inner" container mean?

Reply:
hm yeah that's sound weird, I'll fix that